docs/api/index.html

```diff
@@ -5557,10 +5559,11 @@ <h3 id="planetscale.com/v2.VitessKeyspaceSpec">VitessKeyspaceSpec
 </em>
 </td>
 <td>
-<p>Images are not customizable by users at the keyspace level because version
-skew across the cluster is discouraged except during rolling updates,
-in which case this field is automatically managed by the VitessCluster
-controller that owns this VitessKeyspace.</p>
+<p>Images are inherited from the VitessCluster spec, unless the user has
+specified keyspace-level overrides. Version skew across the cluster is
+discouraged except during rolling updates, in which case this field is
+automatically managed by the VitessCluster controller that owns this
+VitessKeyspace.</p>
```

Comment:
I *think* the version here was more about Vitess than MySQL. I *think* that we're now focusing on MySQL version rather than Vitess version, right?

Reply:
Ah! Yes indeed, just focusing on MySQL.

Hm..with that observation in mind, do you think it would make sense to scope this PR to only allow MySQL image overrides?

Reply:
I went ahead an remove support for changing vttablet, vtorc, myqsld exporter along w/ removing support for setting mariadb images.